exonum/src/node/state.rs

```diff
@@ -743,6 +743,7 @@ impl State {
         // TODO: Destruct/construct structure HeightState instead of call clear. (ECR-171)
         self.blocks.clear();
         self.proposes.clear();
+        self.unknown_proposes_with_precommits.clear();
```

Comment:
Is it possible to write a test covering this change? (As far as I understand, it is this change that triggered the bug.)

Reply:
In theory this is possible but I suppose that this test case will be complex.